sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -70,17 +78,38 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       .where("educationProgram.id = :programId", { programId })
       .andWhere("institutionLocation.id = :locationId", { locationId });
     if (offeringTypes) {
-      offeringsQuery = offeringsQuery.andWhere(
-        "offerings.offeringType in (:...offeringTypes)",
-        {
-          offeringTypes,
-        },
+      offeringsQuery.andWhere("offerings.offeringType in (:...offeringTypes)", {
+        offeringTypes,
+      });
+    }
+    // search offering name
+    if (paginationOptions?.searchName) {
```

Comment:
Is paginationOptions an optional field? if so it can be declared at line 62 as optional

Reply:
`paginationOptions ` is not optional, but don't need null check, removed